### PR TITLE
fix wrong syntax

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2500

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ The final command sets the value of the printer configuration to the modified va
 
 ### Example 3: Set the default paper size of all printers
 ```
-PS C:\>$Printers = Get-Printer * Foreach ($Printer in $Printers){     Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
+PS C:\>$Printers = Get-Printer ; Foreach ($Printer in $Printers){     Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
```
Comment (dariomws) [inline docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md:80]:
```suggestion
PS C:\> $Printers = Get-Printer | Foreach ($Printer in $Printers){Set-PrintConfiguration -PrinterName $Printer.name -PaperSize A4}
```

**Review by dariomws — CHANGES_REQUESTED** [review]

File: docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md (hunk 1 above)
Comment (get-itips) [reply docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md:80]: actually, the ";" is ok here, it allows for multiple statements on a single line

**Review by get-itips — APPROVED** [review]

File: docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md (hunk 1 above)
Comment (dariomws) [reply docset/winserver2019-ps/printmanagement/Set-PrintConfiguration.md:80]: It's true

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**